src/components/MovieTile.tsx

```diff
@@ -0,0 +1,24 @@
+type MovieTileProps = {
+    coverUrl: string
+    title: string
+    year: number
+    genres: string[]
+    cb: () => void
+}
+
+export function MovieTile({
+    coverUrl,
+    title,
+    year,
+    genres,
+    cb,
+}: MovieTileProps) {
+    return (
+        <div onClick={cb}>
```

Comment:
Divs are not clickable elements by default, so, generally speaking, it is not a good practice to assign click callbacks to them. At the very list, you should add `role="button"` to such clickable elements. 
Additional resources: 
https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML
https://dev.to/jakepartusch/how-to-create-a-button-with-a-div-in-react-and-why-you-shouldn-t-1743